From 427e463e942aae700a62d8aa546146f1e27eaac8 Mon Sep 17 00:00:00 2001
From: bloodbowel <177868717+bloodbowel@users.noreply.github.com>
Date: Wed, 21 Aug 2024 22:28:11 -0500
Subject: [PATCH 090/125] Give zombie prisoner variants their full name
 (#75839)

* add zombie to prisoner variant names, shorten to prison zombie

* revert names to "prisoner zombie"
---
 data/json/monsters/zed_prisoner.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/json/monsters/zed_prisoner.json b/data/json/monsters/zed_prisoner.json
index 295145cc9f..dfc51ac94e 100644
--- a/data/json/monsters/zed_prisoner.json
+++ b/data/json/monsters/zed_prisoner.json
@@ -14,7 +14,7 @@
   {
     "id": "mon_zombie_prisoner_brute",
     "type": "MONSTER",
-    "name": { "str": "prisoner brute" },
+    "name": { "str": "prisoner zombie brute" },
     "description": "This muscular zombie wears black and white striped prisoner clothes.",
     "copy-from": "mon_zombie_brute",
     "looks_like": "mon_zombie_brute",
@@ -23,7 +23,7 @@
   {
     "id": "mon_zombie_prisoner_fat",
     "type": "MONSTER",
-    "name": { "str": "fat prisoner" },
+    "name": { "str": "fat prisoner zombie" },
     "description": "This fat zombie wears black and white striped prisoner clothes.",
     "copy-from": "mon_zombie_fat",
     "looks_like": "mon_zombie_fat",
@@ -32,7 +32,7 @@
   {
     "id": "mon_zombie_prisoner_tough",
     "type": "MONSTER",
-    "name": { "str": "tough prisoner" },
+    "name": { "str": "tough prisoner zombie" },
     "description": "This tough-looking zombie wears black and white striped prisoner clothes.",
     "copy-from": "mon_zombie_tough",
     "looks_like": "mon_zombie_tough",
-- 
2.45.2

